configure.in

```diff
@@ -993,6 +993,13 @@ dnl
 dnl more features
 dnl
 
+dnl OML
+
+AC_ARG_ENABLE(oml, [[  --enable-oml  enable OML]], :, enable_oml=no)
```

Comment:
I think this is done

2015-07-30 18:26 GMT+03:00 Eddie Kohler <notifications@github.com>:

> In configure.in
> <https://github.com/kohler/click/pull/206#discussion_r35882953>:
>
> > @@ -993,6 +993,13 @@ dnl
> >  dnl more features
> >  dnl
> >
> > +dnl OML
> > +
> > +AC_ARG_ENABLE(oml, [[  --enable-oml  enable OML]], :, enable_oml=no)
>
> Probably wants to be AC_ARG_WITH (see later for examples)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kohler/click/pull/206/files#r35882953>.
>



-- 
Kostas Choumas
PhD Candidate - University of Thessaly, Greece
Centre for Research and Technology Hellas (CERTH)
T: +30 2421306064
S: kohoumas
E: kohoumas@gmail.com
W: http://web.nitlab.inf.uth.gr/~kohoumas <http://volos.iti.gr/%7Ekohoumas>